src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -1093,34 +1093,17 @@ let Resolve (getVersionsRaw : PackageVersionsFunc, getPreferredVersionsRaw : Pre
 
     let loopTime = DateTime.UtcNow
 
-    let rec fuseConflicts stackpack currentRequirement filteredVersions currentConflict priorConflictSteps conflicts =
+    let rec fuseConflicts stackpack currentRequirement filteredVersions currentConflict priorConflictSteps =
         let currentConflict,stackpack = boostConflicts filteredVersions currentRequirement stackpack currentConflict
-        let matchingStep =
-            let currentNames =
-                conflicts
-                |> Seq.map (fun c ->
-                    c.Graph
-                    |> Set.map (fun (pr:PackageRequirement) -> pr.Name) 
-                    |> Set.add c.Name)
-                |> Set.unionMany
-
-            priorConflictSteps
```

Comment:
@mrinaldi IIRC this was a shortcut that you introduced. I would like to reintroduce it again, but I don't know how.